gsa/src/gmp/models/nvt.js

```diff
@@ -157,6 +149,25 @@ class Nvt extends Info {
     }
     delete ret.cvss_base;
 
+    if (isDefined(ret.solution)) {
+      const solutionType = ret.solution._type;
+      const solutionText = ret.solution.__text;
+      const solutionMethod = ret.solution._method;
+      if (ret.severity === 0) {
+        ret.solution = {
+          type: solutionType === '' ? undefined : solutionType,
+          description: solutionText === '' ? undefined : solutionText,
+          method: solutionMethod === '' ? undefined : solutionMethod,
+        };
+      } else {
+        ret.solution = {
+          type: solutionType,
+          description: solutionText,
+          method: solutionMethod,
+        };
+      }
```

Comment:
Please keep the existing functions for such cases in mind and keep in mind we should have not several meanings for the same thing. No solution type, text, method should mean it has an undefined value.
```suggestion
      ret.solution = {
        type: isEmpty(solutionType) ? undefined : solutionType,
        description:  isEmpty(solutionText) ? undefined : solutionText,
        method: isEmpty(solutionMethod) ? undefined : solutionMethod,
      };
```